apps/tsri/pages/_app.tsx

```diff
@@ -0,0 +1,176 @@
+import {Container, css, CssBaseline, styled, ThemeProvider} from '@mui/material'
```

Comment:
all of that stuff is fresh from master, right? anyway, I'd wait until the other PRs are merged to master, than update this one with master and than merge this one to master